ToolChain/Assembler/Parser/Parser.py

```diff
@@ -632,7 +414,8 @@ def translateTextImmediateToImmediate(self, textImmediate: str=""):
         immediate &= 0xFFFFFFFF  # Maximum immediate value is 32 bits
 
         if validationImmediate != immediate:
-            raise ValueError("Given immediate value is too big, {} received but maxim value is 0xFFFFFFFF".format(hex(validationImmediate)))
+            raise ValueError("Given immediate value is too big, {} received but maxim value is 0xFFFFFFFF".format(
```

Comment:
I don't like how the multi line is presented here. Please make sure that everything is aligned.